app/views/pages/privacy.html.erb

```diff
@@ -4,22 +4,77 @@
+    <p>We keep personal identifiers, contacts and characteristics (for example, name and contact details), email addresses for organisations which may belong to identifiable individuals, phone numbers for organisations which may belong to identifiable individuals, and addresses for events which may belong to identifiable individuals for as long as this information is relevant – meaning as long as a partner, user or calendar this information is associated with is active. ‘Active’ in this sense refers to being a live account which can be logged into by the user, a partner with an associated calendar, or a calendar with a working source.</p>
+    <p>When a user requests to delete their account we will remove all personal identifying information associated with that user account. This does not automatically remove that same individual’s information from partners and calendars. For a user’s information to be removed from a partner or calendar, or for a partner or calendar that user manages to be deleted, this needs to be specified. Once a user account is deleted their information will be removed from PlaceCal and no longer be accessible by any user or staff.</p>
+    <p>Copies of this data may be created by employees of Geeks for Social Change Ltd and Place Health Technology CIC solely for development purposes. This information is destroyed when employees leave the company.</p>
+    
+    <h4>Your data protection rights</h4>
+    <p>Under data protection law, you have rights including:</p>
+    <p><strong>Your right of access</strong> - You have the right to ask us for copies of your personal information.</p>
+    <p><strong>Your right to rectification</strong> - You have the right to ask us to rectify personal information you think is inaccurate. You also have the right to ask us to complete information you think is incomplete.</p>
+    <p><strong>Your right to erasure</strong> - You have the right to ask us to erase your personal information in certain circumstances.</p>
+    <p><strong>Your right to restriction of processing</strong> - You have the right to ask us to restrict the processing of your personal information in certain circumstances.</p>
+    <p><strong>Your right to data portability</strong> - You have the right to ask that we transfer the personal information you gave us to another organisation, or to you, in certain circumstances.</p>
+    <p>You are not required to pay any charge for exercising your rights. If you make a request, we have one month to respond to you.</p>
+    <p>Please contact us at <a href="mailto:support@placecal.org">support@placecal.org</a> if you wish to make a request.</p>
+    
+    <h4>How to complain</h4>
+    <p>If you have any concerns about our use of your personal information, you can make a complaint to us at support@placecal.org</p>
+    <p>You can also complain to the ICO if you are unhappy with how we have used your data.</p>
```

Comment:
@kimadactyl I did a quick search and seems like Geeks have registered with ICO, but Place Health have not. Should I let Peter know? Or someone else?